testing/test_pickle_checkpoint_step.py

```diff
@@ -5,55 +5,125 @@
 from py._path.local import LocalPath
 
 from neuraxle.checkpoints import PickleCheckpointStep
+from neuraxle.hyperparams.space import HyperparameterSamples
 from neuraxle.pipeline import Pipeline
 from neuraxle.steps.util import TapeCallbackFunction, TransformCallbackStep
 
-data_inputs = np.ones((1, 1))
-expected_outputs = np.ones((2, 2))
+EXPECTED_TAPE_AFTER_CHECKPOINT = ["2", "3"]
```

Comment:
What if there are 2 checkpoints in a row? This would perhaps test the behavior of resuming the last step. Or it could be to resume from the first one and not from the last one by manually making the pipeline longer after a first run.

Reply:
+1

Reply:
I will add a test of litteraly two checkpoint steps in a row, but I have some tests with multiple checkpoints.

Reply:
test has to be added inside test_resumable_pipeline not test_pickle_checkpoint